lmdeploy/vl/templates.py

```diff
@@ -239,6 +239,35 @@ def append_image_token(self, prompt, num_images: int):
         return res
 
 
+class MiniCPMVTempateWrapper(VLChatTemplateWrapper):
```

Comment:
What's the chat template of the LLM model in MiniCPM?

Reply:
llama3